api/envoy/config/core/v3/health_check.proto

```diff
@@ -69,15 +69,23 @@ message HealthCheck {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.api.v2.core.HealthCheck.Payload";
 
+    enum Decoder {
+      HEX = 0;
+      UTF8 = 1;
+    }
+
     oneof payload {
       option (validate.required) = true;
 
-      // Hex encoded payload. E.g., "000000FF".
+      // String payload. Default is Hex encoded payload, eg. "000000FF".
```

Comment:
```suggestion
      // String payload. Default is Hex encoded payload, e.g., "000000FF".
```

Reply:
I was under the impression that if `min_len` validator is at least 1, this must have a config, no?

Reply:
Not sure if I follow. Logically, it is fine that this is an empty string I think. But it is under oneof validate.required, so min_len looks redundant. There must be some config there.
```
http_health_check {
  path: "/healthcheck"
  receive {
  }
}
``` 
This will fail.